resources/views/components/user/profile-meta.blade.php

```diff
@@ -43,9 +38,16 @@
     @endif
 </div>
 
-@if ($amIModerator)
+{{-- TODO migrate moderation utilities to Filament and remove this component --}}
+@can('manage', App\Models\User::class)
     <x-user.profile.moderation-tools :targetUser="$user" />
-@endif
+@endcan
+
+@can('manage', App\Models\User::class)
+    @if ($user->isModerated())
+        <x-user.profile.moderation-overview :targetUser="$user" />
+    @endif
+@endcan
```

Comment:
I have a feeling this is somehow tied to my muted dates from the past still being visible, but this section isn't appearing for me.

Reply:
Indeed, I think this is happening because your muted date is in the past (you're not currently muted) and there is no other moderation action on your account. The logic for `isModerated()` is:

```php
public function isModerated(): bool
{
    return
        $this->muted_until?->isFuture() // <-- !!
        || $this->unranked_at !== null
        || $this->banned_at !== null
        || $this->DeleteRequested !== null
    ;
}
```